templates/controllers/modals/submissionMetadata/issueEntryTabs.tpl

```diff
@@ -23,6 +23,21 @@
 	{rdelim});
 </script>
 {if not $hideHelp}<p class="pkp_help">{translate key="issue.manage.entryDescription"}</p>{/if}
+<form id='submissionSettingsRevisionForm' class='pkp_form'>
+	{if $submissionSettingsRevisions|@count > 1}
+		<div id='editPreviousRevision'>
+			{fbvFormSection  for="submissionSettingsRevision"}
+				{fbvElement type="select" name="submissionSettingsRevision" id="submissionSettingsRevision" from=$submissionSettingsRevisions translate=false selected=$currentSubmissionSettingsRevision}
+			{/fbvFormSection}
+		</div>
+	{/if}
+	{if $isPublished}
+		<ul class="checkbox_and_radiobutton">
+			{fbvElement type="checkbox" id="saveAsRevision" name="saveAsRevision" label="submission.saveAsRevision"}
```

Comment:
Here too: I think you don't need name -- it will be automatically taken from the id.